### PR TITLE
Add pyvista.Camera class documentation

### PR DESCRIPTION
**Pull request by tkoyama010**
### Overview

This Pull Request add pyvista.Camera class documentation.

```python
import pyvista as pv
import numpy as np
from pyvista import examples

pv.set_plot_theme("document")

camera = pv.Camera()
near_range = 0.3
far_range = 0.8
camera.clipping_range = (near_range, far_range)
unit_vector = np.array(camera.direction) / np.linalg.norm(
    np.array([camera.focal_point]) - np.array([camera.position])
)

frustum = camera.view_frustum(1.0)

position = camera.position
focal_point = camera.focal_point
line = pv.Line(position, focal_point)

bunny = examples.download_bunny()
xyz = camera.position + unit_vector * 0.6 - np.mean(bunny.points, axis=0)
bunny.translate(xyz)

p = pv.Plotter(shape=(2, 1))
p.subplot(0, 0)
p.add_text("Camera Position")
p.add_mesh(bunny)
p.add_mesh(frustum, style="wireframe")
p.add_mesh(bunny)
p.add_mesh(line, color="b")
p.add_point_labels(
    [
        position,
        camera.position + unit_vector * near_range,
        camera.position + unit_vector * far_range,
        focal_point,
    ],
    ["Camera Position", "Near Clipping Plane", "Far Clipping Plane", "Focal Point"],
    margin=0,
    fill_shape=False,
    font_size=14,
    shape_color="white",
    point_color="red",
    text_color="black",
)
p.camera.position = (1.1, 1.5, 0.0)
p.camera.focal_point = (0.2, 0.3, 0.3)
p.camera.up = (0.0, 1.0, 0.0)
p.camera.zoom(1.4)

p.subplot(1, 0)
p.add_text("Camera View")
p.add_mesh(bunny)
p.camera = camera
p.show()
```
camera_0_0

 
Relate to #945

### Details

- Add `pyvista.Camera` class and API documentation.
- Add `view_frustum` method to `pyvista.Camera` class.

### CODE REVIEW

**Comment by banesullivan** [comment]
This is really great! Thanks, @tkoyama010

File: pyvista/plotting/camera.py [hunk 1]
```diff
@@ -297,8 +298,58 @@ def direction(self):
...
+    def direction(self, direction):
+        """Set the vector from the camera position to the focal point.
+
+        Examples
+        --------
+        >>> import pyvista
+        >>> plotter = pyvista.Plotter()
+        >>> plotter.camera.direction = (0.0, 0.0, -1.0)
+
+        """
```
Comment (banesullivan) [inline pyvista/plotting/camera.py:318]: Needs code?

**Comment by akaszynski** [comment]
Really like these docs.

File: pyvista/plotting/camera.py (hunk 1 above)
Comment (akaszynski) [reply pyvista/plotting/camera.py:318]: Yea, code on setters isn't necessary and won't be shown in sphinx.

Comment (tkoyama010) [reply pyvista/plotting/camera.py:318]: No this setter is not need.

File: pyvista/plotting/camera.py [hunk 2]
```diff
@@ -297,8 +298,58 @@ def direction(self):
         --------
         >>> import pyvista
         >>> plotter = pyvista.Plotter()
-        >>> plotter.camera.direction  # doctest: +SKIP
+        >>> plotter.camera.direction = (0.0, 0.0, -1.0)
+        >>> plotter.camera.direction
```
Comment (tkoyama010) [inline pyvista/plotting/camera.py:302]:
```suggestion
        >>> plotter.camera.direction  # doctest: +SKIP
```

File: pyvista/plotting/camera.py (hunk 1 above)
Comment (tkoyama010) [inline pyvista/plotting/camera.py:318]:
```suggestion
```

File: tests/test_camera.py [hunk 3]
```diff
@@ -97,3 +97,12 @@ def test_view_angle(camera):
 
 def test_direction(camera):
     assert camera.GetDirectionOfProjection() == camera.direction
+    direction = (0.0, 0.0, -1.0)
+    camera.direction = direction
+    assert camera.direction == direction
```
Comment (tkoyama010) [inline tests/test_camera.py:102]:
```suggestion
```

**Comment by tkoyama010** [comment]
@akaszynski @banesullivan Thank you for your comment. It is ready for review.

**Review by tkoyama010 — COMMENTED** [review]
WFM

**Review by banesullivan — APPROVED** [review]
LGTM

**Review by akaszynski — APPROVED** [review]
This is really great.  And nice work seeing that we have to use `jupyter-execute` now that we're not using the `testcode` directive.

**Pull request merged**